server/request/request.go

```diff
@@ -199,6 +201,15 @@ func GetNamespace(ctx context.Context) (string, error) {
 	return "", ErrNamespaceNotFound
 }
 
+func IsHumanUser(ctx context.Context) bool {
```

Comment:
can this be moved as method of request metadata?

Reply:
We will refactor this, once you finish new signup flow.